src/docker.js

```diff
@@ -4,31 +4,30 @@ const mem = require('mem')
 const { request } = require('@octokit/request')
 
 /**
- * Get the base image specified in the FROM line of the provided Dockerfile
- * contents.
+ * Get the base image(s) specified in the FROM line(s) of the provided
+ * Dockerfile contents.
  *
  * @param {string} dockerfileContents
- * @returns {undefined | string}
+ * @returns {string[]}
  */
-const getBaseImageFrom = dockerfileContents => {
+const getBaseImagesFrom = dockerfileContents => {
   if (!dockerfileContents) {
-    return undefined
+    return []
   }
   
-  const fromLine = getFromLine(dockerfileContents)
-  const fromIndex = fromLine.indexOf('FROM')
-  const spaceIndex = fromLine.indexOf(' ', fromIndex)
-  return fromLine.substring(spaceIndex + 1, fromLine.length).trim()
+  const fromLines = getFromLines(dockerfileContents)
```

Comment:
Optionally, I favor something like:
```
    return getFromLines(dockerfileContents)
        .map(getDockerImageFromFromLine)
        .filter(dockerImage => dockerImage !== null)
```

Reply:
Yeah, there are times where chaining methods is helpful/cleaner. In this case, it seems like assigning the result of each step to a named value simplifies the mental task of understanding what the code is doing. I don't have to hold the entire series of operations in my head in order to figure out what the final one is doing; I can instead just evaluate each step on its own, which is easier and less mentally draining. Thanks for the suggestion, though. 👍